Yosemite/Yosemite/Stores/ProductStore.swift

```diff
@@ -170,6 +173,22 @@ private extension ProductStore {
         }
     }
 
+    func searchInCache(siteID: Int64, keyword: String, pageSize: Int, onCompletion: @escaping (Bool) -> Void) {
+        let namePredicate = NSPredicate(format: "name LIKE[c] %@", keyword)
+        let sitePredicate = NSPredicate(format: "siteID == %lld", siteID)
+        let predicate = NSCompoundPredicate(andPredicateWithSubpredicates: [namePredicate, sitePredicate])
```

Comment:
Just to confirm, with this predicate, we're searching for any product name which matches for 0 or more characters, for that specific siteID, is that right?